.github/workflows/positron-python-ci.yml

```diff
@@ -46,11 +46,14 @@ jobs:
           cache: 'npm'
           node-version: ${{ env.NODE_VERSION }}
 
-      - name: Install Python ${{ env.PYTHON_VERSION }}
-        uses: actions/setup-python@v5
+      - name: Install uv and Python
+        uses: astral-sh/setup-uv@v5
```

Comment:
Their [docs](https://docs.astral.sh/uv/guides/integration/github/#setting-up-python) call out that if you've already set up uv it might be faster to use the `actions/setup-python@v5` action due to caching. I just need to test that out, make sure it works with our matrixes, and is actually faster.